pkg/asset/installconfig/powervs/session.go

```diff
@@ -94,30 +87,36 @@ func getPISession() (*ibmpisession.IBMPISession, *UserCredentials, error) {
 			},
 		}, &apikey)
 		if err != nil {
-			return nil, nil, errors.New("Error saving the API_KEY variable")
+			return nil, errors.New("Error finding the API_KEY variable")
```

Comment:
same comment here. and let's be consistent with "finding" or "saving" in the messages. Also I don't think this has anything to do with the env var at this point does it? So just "Error saving API Key." ?